### DISCUSSION TITLE
Intellisense Environments not Automatically Closing

### DISCUSSION THREAD

**Discussion by trevorflahardy** [James-Yu/LaTeX-Workshop#4190] [Q&A]
Hey there,

For some reason, I cannot get IntelliSense to automatically close newly created environments on  VSCode (the documentation calls this the ["**\begin** only approach"](https://github.com/James-Yu/LaTeX-Workshop/wiki/Intellisense#environments)). IntelliSense will recommend the environment name, and when I select it drop the dropdown and hit enter, it will complete the recommendation but fail to enter a closing environment indiciator. 

```tex
\begin{center} % <- Recommended with no problem
\end{center} % <- Will not be created automatically, I have to do it manually
```

In my settings on VSCode, the IntelliSense package and IntelliSense package env are both enabled, and my IntelliSense trigger is set to "{". 

Any help on this is appreciated, if you need more information please do not hesitate to let me know.

**Comment by jlelong** [comment]
I cannot reproduce the issue, so it is hard to help.
